### DISCUSSION TITLE
Optional triangle visualization on main menu and song selection

### DISCUSSION THREAD

**Discussion by KirbyJoe** [ppy/osu#13023] [Ideas]
**Describe the new feature:** 
On old cutting edge versions of osu! which are released in November 2014, the game shows the triangles on background of main menu and song selection, but it's removed due to performance issues. Since the triangles are working fine on osu!lazer, I think it's good to add it as an optional feature.

**Comment by peppy** [comment]
This was removed from the design language as too excessive. May be feasible via skinning in the future.
